src/applications/personalization/profile/components/direct-deposit/BankInfoCNPv2.jsx

```diff
@@ -190,6 +190,13 @@ export const BankInfoCNP = ({
           target="_blank"
           rel="noopener noreferrer"
           href="https://www.va.gov/disability/eligibility/"
+          onClick={() => {
+            recordEvent({
+              event: 'profile-navigation',
+              'profile-action': 'view-link',
+              'profile-section': 'education-benefits',
```

Comment:
This is likely the wrong `profile-section`. [I just asked Ben what we should use for this link](https://github.com/department-of-veterans-affairs/va.gov-team/issues/19352#issuecomment-775276789).

Reply:
Thanks for catching that!